backend/drug_targets.py

```
@@ -0,0 +1,387 @@
+        return []
+    try:
+        mech = new_client.mechanism  # type: ignore[union-attr]
+        results = list(
+            mech.filter(molecule_chembl_id=chembl_id).only(
+                "target_chembl_id",
+                "action_type",
+                "mechanism_of_action",
+                "mechanism_comment",
+            )
+        )
+    except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)